build.sbt

```diff
@@ -80,7 +80,7 @@ lazy val derivation = project
 
 lazy val jsonld = project
   .in(file("modules/jsonld"))
-  .dependsOn(core % "compile->compile;test->test")
+  .dependsOn(core % "compile->compile;test->test", jena)
```

Comment:
why the flip in dependencies?

Reply:
`id` function is using Jena model for finding `@id`

Reply:
right, that makes sense... we said we'll remove this later (post v1.3)